docs/android/submit.md

```diff
@@ -0,0 +1,73 @@
+
+### Checklist for Submitting a PR
+
+When creating a PR, GitHub pre-fills the description with a checklist. Ensure you follow all the steps. Here's an extended checklist to help you:
+
+- ✅ **PR Description**: Provide a clear and complete description of your changes.
+- ✅ **Tests**: Add all necessary tests following our [testing guidelines](testing/introduction).
+- ✅ **Documentation**: Ensure your code is properly documented.
+- ✅ **UI Changes**: Include screenshots if the UI is modified.
+- ✅ **User Documentation**: If user documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/companion.home-assistant).
+- ✅ **Developer Documentation**: If this documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/developers.home-assistant/).
+- ✅ **Builds**: Verify that everything builds properly (app, automotive, wear) locally.
+- ✅ **Best Practices**: Follow the [best practices](bestpractices).
+- ✅ **Code Style**: Adhere to the [code style](codestyle).
+- ✅ **Linting**: Ensure no lint issues are introduced ([linter](linter)).
+
+### Opening a Draft PR
+
+If your PR isn't ready for official review but you'd like feedback, open it in **draft mode**. This will still trigger the CI on every commit and is especially useful when working on CI-related changes.
+
+:::warning
+**Avoid unnecessary CI runs**  
+Running CI workflows consumes significant resources. If your work is incomplete, postpone opening the PR (even in draft mode) unless necessary. Let's be mindful of resource usage and our planet. 🌍 But it doesn't prevent you from pushing regularly to avoid loosing your work.
+:::
+
+### Review Process
+
+#### Who Can Review?
+
+Everyone can comment on your PR since it's public. We encourage contributions through reviews. Reviewing can be quicker than coding, and even a 10-minute review can be valuable.  
+
+If you're not confident in **coding**, you can still help by:  
+
+- Testing the feature by installing the APK (it's available on the Checks tab of the PT).  
+- Providing feedback on UI/UX.  
+- Reporting crashes or bugs.
+
+#### Getting Approval from a Maintainer
+
+Once your PR meets the checklist requirements, wait for a maintainer to review it. Remember, maintainers are volunteers contributing in their free time. Be respectful, patient, and kind.  
+
+Feedback from maintainers will come as:  
+
+- **Comments**: Suggestions or required changes in the code.  
+- **Questions**: Questions about how things are working.
+- **Suggestions**: Improvements or alternative approaches.
+
+### After Receiving Feedback
+
+#### Re-ask for Review
+
+If you've addressed feedback and pushed changes to your PR, you can request a re-review from maintainers. Ensure the CI is green before doing so.
+
+### Merging Your PR
+
+- Keep your PR up to date with the `main` branch.  
```

Comment:
Is this really needed? Update/rebase it before opening your PR or if it's been a long time, but as long as there are no merge conflicts there is not really a reason to do so, it only wastes resources.

Reply:
Well since we are not running tests on main, IMO we can't take the risk to not be up to date on main otherwise the next dev that creates a PR will have to fix the test that fails otherwise he cannot merge.

This only happens if something gets merged that breaks the PR that is going to be merged.

Reply:
Hmm I see your point but don't like a PR with 20 merge commits in it. We could:

 - introduce a test on merging (but flaky)
 - rewrite it more like [the core docs #2](https://developers.home-assistant.io/docs/review-process#speeding-up-the-review-process) as a middle ground

Reply:
Another solution that I've setup in the past is the usage of the [Github merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue). 

It comes with the drawback that when you press the button "add to the queue" it triggers the whole pipeline again even if you are up to date with `main` (if not it is relevant).

The advantage is if you want to merge multiple PR you are sure that when it reach main everything is tested. With the risk of the flakiness still as describe.

I propose we keep the doc like this for now because that is the reality and if we agree I can setup the merge queue and update the doc to explain why we do this.